doc/lua_api.txt

```diff
@@ -5176,105 +5176,134 @@ Definition tables
-    --    maximum send and generate distances, so acts like a telescope.
-    --    Smaller zoomFOV values increase the distance loaded and/or generated.
-    --    Defaults to 15 in creative mode, 0 in survival mode.
-    --    zoom_fov = 0 disables zooming for the player.
+        -- For players only. Zoom FOV in degrees.
+        -- Note that zoom loads and/or generates world beyond the server's
+        -- maximum send and generate distances, so acts like a telescope.
+        -- Smaller zoom_fov values increase the distance loaded/generated.
+        -- Defaults to 15 in creative mode, 0 in survival mode.
+        -- zoom_fov = 0 disables zooming for the player.
+
         eye_height = 1.625,
-    --  ^ For players only. Camera height above feet position in nodes.
-    --    Defaults to 1.625.
+        -- For players only. Camera height above feet position in nodes.
+        -- Defaults to 1.625.
+
         physical = true,
+
         collide_with_objects = true,
-    --  ^ Collide with other objects if physical = true.
+        -- Collide with other objects if physical = true
+
         weight = 5,
-        collisionbox = {-0.5, 0.0, -0.5, 0.5, 1.0, 0.5},
+
+        collisionbox = {-0.5, 0.0, -0.5, 0.5, 1.0, 0.5},  -- Default
         selectionbox = {-0.5, 0.0, -0.5, 0.5, 1.0, 0.5},
-    --  ^ Default, uses collision box dimensions when not set.
-    --  ^ For both boxes: {xmin, ymin, zmin, xmax, ymax, zmax} in nodes from
-    --    object position.
+        -- ^ Uses collision box dimensions when not set.
+        -- For both boxes: {xmin, ymin, zmin, xmax, ymax, zmax} in nodes from
+        -- object position.
+
         pointable = true,
-    --  ^ Overrides selection box when false.
+        -- Overrides selection box when false
+
         visual = "cube" / "sprite" / "upright_sprite" / "mesh" / "wielditem",
-    --  ^ "cube" is a node-sized cube.
-    --  ^ "sprite" is a flat texture always facing the player.
-    --  ^ "upright_sprite" is a vertical flat texture.
-    --  ^ "mesh" uses the defined mesh model.
-    --  ^ "wielditem" is used for dropped items
-    --    (see builtin/game/item_entity.lua).
-    --    For this use 'textures = {itemname}'.
-    --    If the item has a 'wield_image' the object will be an extrusion of
-    --    that, otherwise:
-    --    If 'itemname' is a cubic node or nodebox the object will appear
-    --    identical to 'itemname'.
-    --    If 'itemname' is a plantlike node the object will be an extrusion of
-    --    its texture.
-    --    Otherwise for non-node items, the object will be an extrusion of
-    --    'inventory_image'.
+        -- "cube" is a node-sized cube.
+        -- "sprite" is a flat texture always facing the player.
+        -- "upright_sprite" is a vertical flat texture.
+        -- "mesh" uses the defined mesh model.
+        -- "wielditem" is used for dropped items
+        -- (see builtin/game/item_entity.lua).
```

Comment:
All the lines from here downwards refer to "wielditem" only, this needs to be clear.